packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx

```diff
@@ -147,39 +196,100 @@ export const NftWidget: React.FC<NftWidgetProps> = ({
+            return {
+              title: 'Withdraw your bid to participate',
+              description:
+                'You placed a bid in a previous auction that you can now withdraw to claim back your money and to be able to participate in this auction.',
+            }
+          }
+
+          if (nftStatus.englishTimerState === 'expired' && isOwner && !hasBids) {
+            return {
+              title: 'Auction ended',
+              description: 'This auction has ended, but no one made an offer. You can now remove this NFT from sale.',
+            }
+          }
+          if (nftStatus.englishTimerState === 'expired' && !hasBidFromPreviousAuction && !hasBids && !isOwner) {
+            return {
+              title: 'Auction ended',
+              description:
+                "This auction has ended, but no one made an offer. We're waiting for the NFT owner to cancel the auction.",
+            }
+          }
+          if (
+            nftStatus.englishTimerState === 'expired' &&
+            !hasBidFromPreviousAuction &&
+            hasBids &&
+            !isOwner &&
+            !nftStatus.topBidderIsUser
+          ) {
+            return {
+              title: 'Auction ended',
+              description:
+                'We are waiting for this auction to be settled by the auction winner or the current NFT owner.',
+            }
+          }
+          if (nftStatus.englishTimerState === 'expired' && hasBidFromPreviousAuction) {
+            return {
+              title: 'Withdraw your bid',
+              description: 'You placed a bid in a previous auction that you can now withdraw to claim back your money.',
+            }
+          }
+
+          if (nftStatus.englishTimerState === 'running' && hasBidFromPreviousAuction) {
+            return {
+              title: 'Withdraw your bid to participate',
+              description:
+                'You placed a bid in a previous auction that you can now withdraw to claim back your money and to be able to participate in this auction.',
+            }
+          }
+
+          if (nftStatus.englishTimerState === 'upcoming' && hasBidFromPreviousAuction) {
+            return {
+              title: 'Withdraw your bid to participate',
+              description:
+                'You placed a bid in a previous auction that you can now withdraw to claim back your money and to be able to participate in the upcoming auction.',
+            }
+          }
+
+          return null
+        }
+
+        const infoBannerProps = getInfoBannerProps()
+        return (
           <>
             {nftStatus.topBid ? (
               <NftInfoItem
                 size={size}
                 label="Top bid"
                 content={
                   <>
-                    <JoyTokenIcon size={size === 'small' ? 16 : 24} variant="silver" />
+                    <TopBidderContainer>
+                      <Avatar assetUrl={ownerAvatarUri} size="bid" />
```

Comment:
It's hard to spot, but there is a small outline that comes from avatar, 
![image](https://user-images.githubusercontent.com/51168865/159009566-a572e2e1-f93d-40be-bb77-06a753679e07.png)
We can easily fix this by changing TopBidderTokenContainer zIndex